frontend/src/Lighting.jsx

```diff
@@ -113,11 +97,11 @@ function Lighting() {
         <Entypo name="light-up" size={24} color="black" />
       </View>
       <Subheading>
-        {` Light Color: ${determineColor}`}
```

Comment:
What was the reasoning for removing the display of the color? Because without this display, the stringColor doesn't really do anything.